shared/core/types/proto/param.proto

```diff
@@ -5,14 +5,14 @@ package core;
 option go_package = "github.com/pokt-network/pocket/shared/core/types";
 
 message Param {
-    string Name = 1;
-    string Value = 2;
-    int64  Height = 3;
+    string name = 1;
```

Comment:
Why did you make these lowercase?

Reply:
This was to match the pattern in the other protobuf files and the protobuf documentation - `words_like_this` are turned into `WordsLikeThis` when the go files are generated

Reply:
You are correct!

protoc uppercases the vars when generating the Golang structs